tests/e2e/test_spyre_embeddings.py

```diff
@@ -49,3 +53,59 @@ def test_output(
                               backend=backend,
                               vllm_results=vllm_results,
                               hf_results=hf_results)
+
+
+@pytest.mark.parametrize("warmup_shapes", [
+    (64, 1),
+    (64, 2),
+    (64, 4),
+])  # (prompt_length/batch_size)
+@pytest.mark.parametrize("backend", get_spyre_backend_list())
+@pytest.mark.parametrize("model", get_spyre_model_list(isEmbeddings=True))
+@pytest.mark.parametrize("vllm_version", ["V0", "V1"])
+def test_scheduling_invariance(
+    model,
+    backend,
+    warmup_shapes,
+    vllm_version,
+) -> None:
+
+    os.environ["VLLM_SPYRE_DYNAMO_BACKEND"] = backend
+    os.environ['VLLM_USE_V1'] = "1" if vllm_version == "V1" else "0"
+
+    prompts = get_chicken_soup_prompts(4)
+    reference_embeds = st_embeddings(model, prompts)
+
+    vllm_model = LLM(model=model,
+                     task="embed",
+                     tokenizer=model,
+                     max_model_len=256,
+                     block_size=256,
+                     tensor_parallel_size=1)
+
+    def chunk_embeds(step):
```

Comment:
Wouldn't it make more sense to name this function `batch_embeds` instead of `chunk_embeds`? given that this function embed 1 prompt, then 2 prompts, then 4 prompts.